scripts/validate-index.ts

```diff
@@ -33,6 +35,7 @@ async function run() {
         if (!isValid) {
             validate.errors?.forEach((error) => reportError(file, error));
             isError = true;
+            console.log(`${orgIndex} ${file}`)
```

Comment:
needed for testing?

Reply:
Yup, a leftover after testing. Good catch!